Settings.StyleCop

```diff
@@ -117,6 +163,61 @@
             <BooleanProperty Name="Enabled">False</BooleanProperty>
           </RuleSettings>
         </Rule>
+        <Rule Name="StaticElementsMustAppearBeforeInstanceElements">
+          <!-- it's OK to have static members after instance members -->
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+      </Rules>
+      <AnalyzerSettings />
+    </Analyzer>
+    <Analyzer AnalyzerId="StyleCop.CSharp.LayoutRules">
+      <Rules>
+        <Rule Name="ClosingCurlyBracketMustBeFollowedByBlankLine">
+          <!-- it's OK to not have a blank line after '}' -->
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+        <Rule Name="ElementsMustBeSeparatedByBlankLine">
+          <!-- it's OK to not have a blank line between adjacent members.
+               think about the 'Visit*' methods in Ast visitor class. -->
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+        <Rule Name="ElementMustNotBeOnSingleLine">
+          <!-- it's OK to have simple member on a single line.
+               think about the 'Visit*' methods in Ast visitor class. -->
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+        <Rule Name="StatementMustNotBeOnSingleLine">
+          <!-- it's OK to have a statement on a single line. -->
```

Comment:
I think we use only one very rare related pattern like `if ( … ) { … }`. 
So I'd not turn out the rule.

Reply:
There are 712 violations of this rule, and yes, most are that `if` pattern. Some `catch { .. }` too.
I don't think it's worthwhile to fix them.